v22.1/show-trace.md

```diff
@@ -197,19 +190,76 @@ In this example, we use two terminals concurrently to generate conflicting trans
+    --------------------------------------+-----------------+---------------------------------------------------------------+------------------------------------------------------------+---------------------------------+-----------------------------------+-------
+      2021-04-13 18:55:43.094235+00:00:00 | 00:00:00        | === SPAN START: session recording ===                         |                                                            |                                 | session recording                 |    0
+      2021-04-13 18:55:43.094244+00:00:00 | 00:00:00.000009 | === SPAN START: sync= ===                                     |                                                            |                                 | sync                              |    1
+      2021-04-13 18:55:43.094286+00:00:00 | 00:00:00.000051 | [NoTxn pos:43] executing Sync                                 | [n1,client=127.0.0.1:60668,hostssl,user=root]              | sql/conn_executor.go:1478       | sync                              |    1
+      2021-04-13 18:55:43.094436+00:00:00 | 00:00:00.000201 | === SPAN START: exec stmt ===                                 |                                                            |                                 | exec stmt                         |    2
+      2021-04-13 18:55:43.094473+00:00:00 | 00:00:00.000238 | [NoTxn pos:44] executing ExecStmt: SHOW LAST QUERY STATISTICS | [n1,client=127.0.0.1:60668,hostssl,user=root]              | sql/conn_executor.go:1478       | exec stmt                         |    2
+      2021-04-13 18:55:43.094486+00:00:00 | 00:00:00.000251 | executing: SHOW LAST QUERY STATISTICS in state: NoTxn         | [n1,client=127.0.0.1:60668,hostssl,user=root]              | sql/conn_executor_exec.go:75    | exec stmt                         |    2
+      2021-04-13 18:55:43.094497+00:00:00 | 00:00:00.000262 | === SPAN START: sync ===                                      |                                                            |                                 | sync                              |    3
       ...
-    (200 rows)
     ~~~
 
+## Route traces to a third-party collector
+
+You can configure CockroachDB to send traces to a third-party collector. CockroachDB supports [Jaeger](https://www.jaegertracing.io/), [Zipkin](https://zipkin.io/), and any trace collector that can ingest traces over the standard OTLP protocol. Enabling tracing also activates all the log messages, at all verbosity levels, as traces include the log messages printed in the respective trace context.
+
+{{site.data.alerts.callout_info}}
+Enabling full tracing is expensive both in terms of CPU usage and memory footprint, and is not suitable for high throughput  production environments.
```

Comment:
There appear to be two spaces between 'throughput' and 'production' but maybe it's just the web UI I'm looking at